### DISCUSSION TITLE
Carat icon of select input

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#39297] [Q&A]
how to change carat icon of form select class in bootstrap 5? 
or,
How to change the default dropdown img of select input in bootstrap 5?

**Comment by nkdas91** [comment]
Hi @tilak681481,

You need to update the Select indicator SVG in the following SCSS files:

https://github.com/twbs/bootstrap/blob/ce114117a8710016022c40e717cb1f3ec95a5d7f/scss/_variables.scss#L1014-L1015

https://github.com/twbs/bootstrap/blob/ce114117a8710016022c40e717cb1f3ec95a5d7f/scss/_variables-dark.scss#L64-L65
